packages/styles/alert.css

```diff
@@ -25,8 +32,26 @@
   display: flex;
   justify-content: center;
   background-color: transparent;
+  padding: 0 var(--space-large) var(--space-large);
```

Comment:
I checked with @dequelabs/cauldron-design on this and this can inherit from `--dialog-padding`.